tests/test_mathematical.py

```diff
@@ -1042,23 +1043,46 @@ def test_sum_empty_out(dtype):
 
 
 @pytest.mark.parametrize(
-    "shape", [(), (1, 2, 3), (1, 0, 2), (10), (3, 3, 3), (5, 5), (0, 6)]
+    "shape",
+    [
+        (),
+        (1, 2, 3),
+        (1, 0, 2),
+        (10,),
+        (3, 3, 3),
+        (5, 5),
+        (0, 6),
+        (10, 1),
+        (1, 10),
+    ],
 )
 @pytest.mark.parametrize(
     "dtype_in", get_all_dtypes(no_complex=True, no_bool=True)
 )
 @pytest.mark.parametrize(
     "dtype_out", get_all_dtypes(no_complex=True, no_bool=True)
 )
-def test_sum(shape, dtype_in, dtype_out):
-    a_np = numpy.ones(shape, dtype=dtype_in)
-    a = dpnp.ones(shape, dtype=dtype_in)
-    axes = [None, 0, 1, 2]
+@pytest.mark.parametrize("transpose", [True, False])
+@pytest.mark.parametrize("keepdims", [False])
```

Comment:
Setting `keepdims` to `True` results in sum tests to fail.
See details: https://github.com/IntelPython/dpnp/issues/1487

Reply:
the issue is resolved, could you please add the testing with `keepdims=True`

Reply:
done